src/containers/App/index.tsx

```diff
@@ -1,17 +1,203 @@
+  };
+  
+  const spacer: React.CSSProperties = {
+     flex: "1", textAlign: "center", color: "white" 
+  };
+
+  const toolbar_navigation_items: React.CSSProperties = {
+    display: "flex",
+    justifyContent: "space-evenly",
+    marginTop: "0.5rem"
+  }
+
+  const toolbar_navigation_itemsp: React.CSSProperties = {
+      color: "white",
+      textDecoration: "none",
+      listStyle: "none",
+      display: "flex",
+      paddingBottom:"10px"
+  };
+
+  const man_icon: React.CSSProperties = {
+    listStyle: "none",
+    display: "block",
+    paddingLeft: "22px",
+    alignItems: "center",
+    height: "30px",
+    paddingRight: "2px"
+  };
+
+  const button_man_icon: React.CSSProperties = {
+    top: "25px",
+    right: "20px",
+    borderRadius: "2rem",
+    cursor: "pointer",
+    outline: "none",
+  };
+
+    const button_logout: React.CSSProperties = {
+      paddingLeft: "20px",
+      paddingTop: "8px",
+      height: "5px"
+    };
+
+    const button_logout1: React.CSSProperties = {
+      backgroundColor: "#3f51bf",
+      border: "none",
+      color: "white",
+      padding: "7px",
+      textAlign: "center",
+      textDecoration: "none",
+      fontSize: "11px",
+      margin: "4px 2px",
+      cursor: "pointer",
+      borderRadius: "1rem",
+      outline: "none"
+    };
+
+
+
+
+
+    const sidebar: React.CSSProperties = {
+      height: "100%",
+      width: "65px",
+      position: "fixed",
+      top: "0",
+      left: "0",
+      backgroundColor: "#3f51bf",
+      transition: "0.4s",
+      overflowX: "hidden",
+      paddingTop: "100px",
+      whiteSpace: "nowrap"
+    };
+    const sidebara: React.CSSProperties = {
+      padding: "8px 8px 8px 20px",
+      textDecoration: "none",
+      fontSize: "25px",
+      color: "white",
+      display: "block"
+    };
+    const icons: React.CSSProperties = {
+      verticalAlign: "middle",
+      marginRight: "23px"
+    };
+    const icontext: React.CSSProperties = {
+      verticalAlign: "middle"
+    };
+
+
+    
 interface Props {
   signOut: () => void;
+  
+}
+
+ function mousehover(e: any) {
+    e.target.style.fontSize= '26px'
+ }
+
+ function mouseout(e: any) {
+  e.target.style.fontSize= '25px'
+}
+
+function displaybutton() {
+  var x = document.getElementById("demo");
+  if (x.style.display === "block") {
+    x.style.display = "none";
+  } else {
+    x.style.display = "block";
+    x.style.color= "white";
+  }
 }
 
-const App: React.FC<Props> = ({ signOut }) => {
+function testIn() {
+  console.log("hovering in sidebar");
+  document.getElementById("mySidebar").style.width = "200px";
+}
+
+function testOut() {
+  console.log("hovering outside sidebar");
+  document.getElementById("mySidebar").style.width = "65px";
+}
+  
+const App: React.FC<Props> = (
+  { signOut }) => {
   return (
-    <div>
-      Timesheets
-      <p>Hey! Let's clock in.</p>
-      <button onClick={signOut}>Log out</button>
-    </div>
-  );
+        <div>
```

Comment:
We should consider making this into smaller components and importing them here. This component is getting large. You could have a header component, nav component, etc.